README.md

```diff
@@ -44,54 +44,54 @@ You are in charge of your data and customizations.
 <p align="center"><img src="https://github.com/AppFlowy-IO/appflowy/blob/main/doc/imgs/howtostar.gif" alt="AppFlowy Github - how to star the repo" width="100%" /></p>
 
 ## Getting Started with development
+
 Please view the [documentation](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy) for OS specific development instructions
 
 ## Roadmap
 
-- [AppFlowy Roadmap ReadMe](https://appflowy.gitbook.io/docs/essential-documentation/roadmap)
-- [AppFlowy Public Roadmap](https://github.com/orgs/AppFlowy-IO/projects/5/views/12)
-
+* [AppFlowy Roadmap ReadMe](https://appflowy.gitbook.io/docs/essential-documentation/roadmap)
+* [AppFlowy Public Roadmap](https://github.com/orgs/AppFlowy-IO/projects/5/views/12)
```

Comment:
These two are identical in practice, but what's the reason behind this change?

Reply:
It's a markdown linting convention. 

The convention for unordered list:

```
* Item 1
  + Item 2
    - Item 3
  + Item 4
* Item 4
  + Item 5
```

[See Markdownlint](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md004---unordered-list-style)

Reply:
@richardshiue what do you think? should we follow this linting or just do as we like? I didn't see any markdownlint config in this repo, so I think, it should be declared for consistency.

Reply:
Interesting, I didn't know this existed. I personally don't have a preference so let's see what the maintainers say. Perhaps we can add the Github Action fort this as well. Apart from this, LGTM.